### PR TITLE
Add missing community docs

### PR DESCRIPTION
**Pull request by jankapunkt**
This adds the missing community docs

- contribution guidelines
- code-of-conduct
- security

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+- translation (i18n), language, grammar and typos
+
+Other contributions can be done here:
+- [user documentation](https://github.com/openqda/user-docs))
+- [report bugs and issues](https://github.com/openqda/openqda/issues)
+- [discuss new features, ideas or questions](https://github.com/orgs/openqda/discussions)
+
+## Why is this guide important?
+
+There are a few conventions and pattern, for which we will take
```
Comment (hohse) [inline CONTRIBUTING.md:44]: patterns

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+```shell
+$ get checkout main
+$ git pull --ff-only
+```
+
+If the pull cannot "fast-forward", then you have remains of changes on your
+current branch. Please remove them and make sure your `main` branch is "clean".
+
+> Note: if you have forked the repository and your fork is much behind the main repository
+then you need to sync them: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
```
Comment (hohse) [inline CONTRIBUTING.md:78]: I think there should also be a > in front of the line.

**Review by hohse — APPROVED** [review]
I reviewed as best as I could.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jankapunkt) [reply CONTRIBUTING.md:44]: thanks will do

File: CONTRIBUTING.md (hunk 2 above)
Comment (jankapunkt) [reply CONTRIBUTING.md:78]: thanks will do

**Pull request merged**